MashUp-iOS/MashUp-iOS/Source/Screen/Setting/SettingViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SettingViewController.swift
+//  MashUp-iOS
+//
+//  Created by 이문정 on 2022/07/15.
+//  Copyright © 2022 Mash Up Corp. All rights reserved.
+//
+
+import UIKit
+import MashUp_Core
+import MashUp_UIKit
+import RxSwift
+
+final class SettingViewController: BaseViewController {
+    private let navigationBar: MUNavigationBar = MUNavigationBar(frame: .zero)
+    private let settingMenuStackView: UIStackView = UIStackView()
+    private let snsLeftStackView: UIStackView = UIStackView()
+    private let snsRightStackView: UIStackView = UIStackView()
+    private let snsContainerStacView: UIStackView = UIStackView()
+    private let logoutButton: BaseView = SettingMenuView(title: "로그아웃", titleColor: .gray800)
+    private let withdrawalButton: BaseView = SettingMenuView(title: "회원탈퇴", titleColor: .red500)
+    private let facebookButton: BaseView = SNSButton(snsType: .facebook)
+    private let instagramButton: BaseView = SNSButton(snsType: .instagram)
+    private let tistoryButton: BaseView = SNSButton(snsType: .tistory)
+    private let youtubeButton: BaseView = SNSButton(snsType: .youtube)
+    private let mashUpHomeButton: BaseView = SNSButton(snsType: .home)
+    private let recruitButton: BaseView = SNSButton(snsType: .recruit)
+    private let disposeBag: DisposeBag = DisposeBag()
+    
+    override func viewDidLoad() {
+        setupUI()
```

Comment:
```suggestion
        super.viewDidLoad()
        setupUI()
```
당장은 문제가 없을 수 있지만 override할때는 super의 메서드를 호출해주시면 좋을 것 같아요 🙇‍♂️